### ISSUE TITLE
Senti-GAN loss function -ve sign

### ISSUE THREAD

**Issue by anindyasdas** [williamSYSU/TextGAN-PyTorch#54]
The loss in the code (batchPGLoss) for sentigan is as follows:
```
loss = -torch.sum(pred * (1 - reward))
```
why two negative signs? if (1-reward) =Penalty, and we are then negating, doesn't that mean we are minimizing the **negated penalty*prediction** or maximizing the penalty*prediction. In the paper the 

![image](https://user-images.githubusercontent.com/50780535/117251216-f9be6f80-ae61-11eb-840d-4ffcbc3e63b0.png)
